src/pages/InterviewSchedulingPage/components/InterviewScheduler.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import ScheduleSelector from 'react-schedule-selector';
+
+interface InterviewSchedulerProps {
+  numDays?: number;
+  minTime?: number;
+  maxTime?: number;
+  startDate?: Date;
+  reset?: boolean;
+  setDirtyCallback?: () => void;
+  selectedScheduleCallback?: (arg: Date[]) => void;
+}
+
+function InterviewScheduler({
+  numDays = 5,
+  minTime = 11,
+  maxTime = 21,
+  startDate = new Date(),
+  reset = false,
+  setDirtyCallback = () => {}, // eslint-disable-line @typescript-eslint/no-empty-function
+  selectedScheduleCallback = () => {}, // eslint-disable-line @typescript-eslint/no-empty-function
```

Comment:
same comment as above, would probably be nicer if we only passed in 1 function as props